dbt_subprojects/daily_spellbook/models/_metrics/transfers/metrics_transfers_daily.sql

```diff
@@ -61,20 +63,24 @@ with raw_transfers as (
     /*
     - create one column for transfer amount received, one column for transfer amount sent
     */
-    select 
-        blockchain
-        , block_date
-        , tx_hash
-        , address
-        , sum(case when transfer_direction = 'sent' then transfer_amount_usd else 0 end) as transfer_amount_usd_sent
-        , sum(case when transfer_direction = 'received' then transfer_amount_usd else 0 end) as transfer_amount_usd_received
+    select
+        t.blockchain
+        , t.block_date
+        , t.tx_hash
+        , coalesce(l.owner_key, cast(t.address as varchar)) as address_owner
```

Comment:
this field isn't used in final spell, only steps within CTEs to group by for aggregating totals accurately. that's why we're less concerned on the cast to varchar on address.